testsuite/tests/kuadrant/identical_hostname/test_auth_on_two_routes.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tests behavior of using two HTTPRoutes declaring the same hostname related to AuthPolicy.
+https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/auth.md#limitation-multiple-network-resources-with-identical-hostnames
+(the first topology mentioned there)
+"""
+
+import pytest
+
+from testsuite.policy.authorization.auth_policy import AuthPolicy
+from testsuite.utils import has_condition
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+@pytest.fixture(scope="class")
+def authorization2(request, kuadrant, route2, authorization_name2, openshift, label):
+    """2nd Authorization object (In case of Kuadrant AuthPolicy)"""
+    if kuadrant:
```

Comment:
Not needed, as is is marked as `kuadrant_only`. Also if it ever happened it would fail with an error

Reply:
You're right, that was copy/paste from `authorization` fixture but it is not needed for `authorization2` fixture. Will fix it. Thanks!

Reply:
Done